### PR TITLE
(Credit to Vorecode) God Idols for Chaplain

### PR DESCRIPTION
**Pull request by MrFagetti**
fsadasf

## About The Pull Request

Adds religious idols, ported from Vorestation code

NOTE:

Currently you cannot rename the idols, only change the desc and icon (both are tied together). Right clicking will open the drop down and show the option. I feel it would be better to have it Alt+click but I'm still fairly new to coding tbh.

Also you can change the idol multiple times

## Why It's Good For The Game

More Chaplain RP stuff, plus having lil statuettes has been a want for me as Chaplain for years.

## Changelog
:cl:
add: Religious Idols from Vorestation
add: several new idols, some racial and some general purpose
/:cl:

### CODE REVIEW

**Comment by MrFagetti** [comment]
Since this was effectively just a copy paste, I can say this is ready for TM atleast.

I had to remove the non-functional rename part since it was throwing a bitch fit about me commenting it out.

File: modular_lumos/code/game/objects/items/godfigures.dm [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+		else if(options[choice] == "skull")
+			desc = "A golden holy figure of a humanoid skull."
+		else if(options[choice] == "devil")
+			desc = "A painted holy figure of a seated humanoid goat with wings."
+		else if(options[choice] == "sun")
+			desc = "A holy figure of a star."
+		else if(options[choice] == "moon")
+			desc = "A holy figure of a small planetoid."
+		else if(options[choice] == "lizrobe")
+			desc = "A painted holy figure of a plain looking Uthani in a robe."
```
Comment (coldud13) [inline modular_lumos/code/game/objects/items/godfigures.dm:117]: yandev moment

Comment (MrFagetti) [reply modular_lumos/code/game/objects/items/godfigures.dm:117]: lol blame the vore cooders

**Comment by MrFagetti** [comment]
TM it you fat aussie

**Comment by MrFagetti** [comment]
Goblin_Drip_Big

**Comment by coldud13** [comment]
so how are these supposed to be obtained in game?
also i only changed a few of the idols to actual objects, the rest still needs to be converted over sometime

**Pull request merged**